src/accelerate/utils/imports.py

```diff
@@ -142,6 +142,10 @@ def is_torch_xla_available(check_is_tpu=False, check_is_gpu=False):
     return True
 
 
+def is_torchao_available():
+    return _is_package_available("torchao")
```

Comment:
would it be possible to require torchao v0.6.1 or later?  It should work with earlier versions but a more recent version is ideal for better performance.  If needed I can look up the exact earliest version which we can expose - lmk.

Reply:
I see no issue with that, especially as it's a young library too which makes our lives easier since it's not in accelerate yet!